packages/syft/src/syft/core/tensor/lazy_repeat_array.py

```diff
@@ -262,6 +262,10 @@ def sum(self, *args: Tuple[Any, ...], **kwargs: Any) -> lazyrepeatarray:
         res = np.array(self.to_numpy().sum(*args, **kwargs))
         return lazyrepeatarray(data=res, shape=res.shape)
 
+    def mean(self, *args: Tuple[Any, ...], **kwargs: Any) -> lazyrepeatarray:
+        res = np.array(self.to_numpy().mean(*args, **kwargs))
+        return lazyrepeatarray(data=res, shape=res.shape)
```

Comment:
We discussed this in our 1:1- but let's take advantage of the fact that we can predict ahead of time what the min_val and max_val of the mean would be :)

Reply:
I _think_ I've got this at least started down the right path, where we'd start passing new min/max properties in the constructor method. Let's chat more because I think I'm still missing something in how this should be implemented.